hack/release/RELEASING.md

```diff
@@ -180,18 +180,48 @@ When starting development on a new minor release, the first step is to create a
 
 ## 4. Performing a release
 
-### 4.a Build and publish the repository in Semaphore 
+### 4.a Create a temporary branch for this release against origin
+
+1. Create a new branch based off of `release-vX.Y`.
+
+   ```
+   git checkout release-vX.Y && git pull origin release-vX.Y
```

Comment:
nit: I'm not sure what the norm is, but I am not sure if every user has their `origin` git remote set to reference `github.com/projectcalico/calico`. Maybe we want to mention that it's against whatever the appropriate git remote or just say that the instructions assume `origin` points towards `github.com/projectcalico/calico`?

Reply:
At the top of the doc there is a line that says this in the prereqs:

> Finally, the release process assumes that your repos are checked out with name origin for the git remote for the main Calico repo.